content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-11.md

```diff
@@ -194,7 +194,7 @@ To keep `kube-dns`, pass `--feature-gates=CoreDNS=false` to `kubeadm upgrade app
 
     ```shell
     kubectl drain ip-172-31-85-18
-    node "ip-172-31-85-18" cordoned
+    node/ip-172-31-85-18 cordoned
```

Comment:
@tanshanshan have you verified that the output is in this format against a 1.10.x cluster? I'm not sure about this.

Reply:
It describes an upgrade process and I think here the command is executed before upgrading.

Reply:
no, starting from 1.11